From e8164cddd81628bb73eded92800a07217e974145 Mon Sep 17 00:00:00 2001
From: Eyal Royee <eyalroyee@gmail.com>
Date: Sat, 18 Sep 2021 15:54:30 +0300
Subject: [PATCH] Changed EXTRAVERSION variable in the main Makefile to
 "-eudyptula-challenge-solution-3"

Signed-off-by: Eyal Royee <eyalroyee@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9b6c90eed5e9..26ae3a270822 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 10
 SUBLEVEL = 7
-EXTRAVERSION =
+EXTRAVERSION = -eudyptula-challenge-solution-3
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
-- 
2.25.1

